infrastructure/cdn-in-a-box/docker-compose.totest.yml

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What do you think about renaming this to docker-compose.traffic-ops-test.yml?

Reply:
Sure